server/src/sql-tools/from-database/readers/extension.reader.ts

```diff
@@ -0,0 +1,14 @@
+import { DatabaseReader } from 'src/sql-tools/from-database/readers/type';
+
+export const readExtensions: DatabaseReader = async (schema, db) => {
+  const extensions = await db
+    .selectFrom('pg_catalog.pg_extension')
+    // .innerJoin('pg_namespace', 'pg_namespace.oid', 'pg_catalog.pg_extension.extnamespace')
+    // .where('pg_namespace.nspname', '=', schemaName)
```

Comment:
What about this?

Reply:
I don't remember it was just there before lol